recipes/theseus-ai/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "theseus-ai" %}
+{% set version = "0.1.3" %}
+{% set build = 0 %}
+
+{% set processor = "cpu" if cuda_compiler_version == "None" else "gpu" %}  # [not osx]
+{% set processor = "cpu" %}  # [osx]
+
+# Prioritize gpu build if cudatoolkit can be installed (through __cuda virtual package)
+{% set build = build + 100 if processor == "gpu" else build %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/theseus-ai-{{ version }}.tar.gz
+  sha256: 150702236782c76815ad349d2089ef801469a9a8073610da77fe4b89a11ad5d1
+
+build:
+  number: {{ build }}
+  skip: true  # [(osx and cuda_compiler_version != "None") or win]
+  missing_dso_whitelist:
+    # pytorch builds put the libs in the wrong place, but hard to unbundle, see
+    # https://github.com/conda-forge/staged-recipes/pull/17082#issuecomment-1189149813
+    - "*/libc10{{ SHLIB_EXT }}"
+    - "*/libc10_cuda{{ SHLIB_EXT }}"
+    - "*/libtorch{{ SHLIB_EXT }}"
+    - "*/libtorch_cpu{{ SHLIB_EXT }}"
+    - "*/libtorch_cuda{{ SHLIB_EXT }}"
+    - "*/libtorch_python{{ SHLIB_EXT }}"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}         # [not osx and cuda_compiler_version != "None"]
+  host:
+    - python
+    - pytorch
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy >=1.5.3
+    - scikit-sparse >=0.4.5
+    - pytest >=6.2.1
```

Comment:
Awkward to see `pytest` as a run dependency. Is this intentional?

Reply:
It wasn't - thanks for the catch!